components/brave_new_tab_ui/components/default/page/index.tsx

```diff
@@ -26,17 +26,21 @@ interface HasImageProps {
   colorForBackground?: string
 }
 
+interface HasSponsoredRichMediaProps {
+  hasSponsoredRichMediaBackground: boolean
+}
+
```

Comment:
I don't think its worth adding a new interface for this - it just makes all the type definitions more complicated.

Just add the property to `HasImageProps`

(these two interfaces are always used together)

Reply:
Fixed, thanks!